src/sagemaker/lineage/artifact.py

```diff
@@ -54,31 +57,38 @@ class Artifact(_base_types.Record):
         tags (List[dict[str, str]]): A list of tags to associate with the artifact.
         creation_time (datetime): When the artifact was created.
         created_by (obj): Contextual info on which account created the artifact.
+        last_modified_time (datetime): When the artifact was last modified.
+        last_modified_by (obj): Contextual info on which account created the artifact.
     """
 
-    artifact_arn = None
-    artifact_name = None
-    artifact_type = None
-    source = None
-    properties = None
-    tags = None
-    creation_time = None
-    created_by = None
-    last_modified_time = None
-    last_modified_by = None
-
-    _boto_create_method = "create_artifact"
-    _boto_load_method = "describe_artifact"
-    _boto_update_method = "update_artifact"
-    _boto_delete_method = "delete_artifact"
-
-    _boto_update_members = ["artifact_arn", "artifact_name", "properties", "properties_to_remove"]
+    artifact_arn: str = None
+    artifact_name: str = None
+    artifact_type: str = None
+    source: object = None
+    properties: dict = None
+    tags: list = None
+    creation_time: datetime = None
+    created_by: object = None
+    last_modified_time: datetime = None
+    last_modified_by: object = None
+
+    _boto_create_method: str = "create_artifact"
+    _boto_load_method: str = "describe_artifact"
+    _boto_update_method: str = "update_artifact"
+    _boto_delete_method: str = "delete_artifact"
+
+    _boto_update_members = [
+        "artifact_arn",
+        "artifact_name",
+        "properties",
+        "properties_to_remove",
+    ]
 
     _boto_delete_members = ["artifact_arn"]
 
     _custom_boto_types = {"source": (_api_types.ArtifactSource, False)}
 
-    def save(self):
+    def save(self) -> object:
```

Comment:
similar comment as above, can we make this a more specific type?

Reply:
Changed